source/scripts/storage/storage.js

```diff
@@ -0,0 +1,113 @@
+//storage {
+//   categories: {cat1:[names],cat2:[names]}
+//   savedLists: {list1:[names],list2:[names]}
+//   recipeName1: json object
+//   recipeName2: json object
+// }
+
+// store recipe to local storage
+// add recipe name to category
+// input: category of api fetch, array of recipe json object
+export function storeRecipeData(category, recipeArray) {
+  try {
+    // store recipe
+    for (let i = 0; i < recipeArray.length; i++) {
+      localStorage.setItem(
+        recipeArray[i]["title"],
```

Comment:
i think we should be using recipe id for the key (guaranteed to be unique)

Reply:
Good idea